felix/config/param_types_test.go

```diff
@@ -61,6 +61,27 @@ var _ = DescribeTable("CIDR list parameter parsing",
 	Entry("Mix of IP and CIDRs", "1.1.1.1/24, 2.2.2.2", []string{"1.1.1.0/24", "2.2.2.2/32"}, true),
 )
 
+var _ = DescribeTable("Port list parameter parsing",
+	func(raw string, expected []config.ProtoPort, expectSuccess bool) {
+		p := config.PortListParam{config.Metadata{
+			Name: "PortLists",
+		}}
+		actual, err := p.Parse(raw)
+		if expectSuccess {
+			Expect(err).To(BeNil())
+			Expect(actual).To(Equal(expected))
+		} else {
+			Expect(err).To(BeNil())
+		}
+	},
+	Entry("Empty", "", nil, true),
```

Comment:
Interesting, I would have expected an empty string to be set the default - but I guess we have logic to detect "unset" vs "set but empty"? 

If that's the case, I wonder if the right fix is just in documentation to set `""` instead of `"[]"` - WDYT?

Reply:
It doesn't really, based on the env var syntax. 

`tcp:22,udp:1.1.1.1:22,3033` - this is a valid value, and it has no `[` or `]` in it at all!

It's a valid YAML representation, but this isn't YAML we're talking about.

Reply:
Better to fix the docs.

Reply:
It makes sense to set the value as ```[]``` in the felix config but when configured as an env var it must be set to "".